Objects/typevarobject.c

```diff
@@ -1914,7 +1914,16 @@ typealias_alloc(PyObject *name, PyObject *type_params, PyObject *compute_value,
         return NULL;
     }
     ta->name = Py_NewRef(name);
-    ta->type_params = Py_IsNone(type_params) ? NULL : Py_XNewRef(type_params);
+    if (
+        type_params == NULL
```

Comment:
Can it ever be NULL? Can't hurt to have this check though.

Reply:
Old code assumed this path with `Py_XNewRef`, so I just followed the same style.